content/en/docs/getting-started/_index.html

```diff
@@ -0,0 +1,266 @@
+      <br><br>
+      2. At the shell prompt, execute a `powershell.exe` command to download
+          and install the `choco` binary and set the installation path so that
+          the binary can be executed:
+
+{{<highlight sh>}}
+    @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+{{</highlight>}}
+
+      3.  Install Jenkins X using Chocolatey:
+      
+{{<highlight sh>}}
+  choco install jenkins-x
+{{</highlight>}}
+      
+      Upgrading Jenkins X via Chocolatey
+      
+{{<highlight sh>}}
+choco upgrade jenkins-x
+{{</highlight>}}
+      
+      
+      <h3> Install using Scoop</h3>
+      
+      <div class="alert alert-dismissible alert-info">
+        <button type="button" class="close" data-dismiss="alert">&times;</button>
+        <strong>TIP:</strong> 
+        If you use <a href="https://scoop.sh" class="alert-link">Scoop</a>, then there is a <a href="https://github.com/lukesampson/scoop/blob/master/bucket/jx.json" class="alert-link">manifest available</a>
+      </div>
+      
+        Install the `jx` binary
+      
+{{<highlight sh>}}
+scoop install jx
+{{</highlight>}}
+      
+        Upgrade the `jx` binary
+      
+{{<highlight sh>}}
+scoop update jx
+{{</highlight>}}
+        </div>
+      </div>
+    </div>
+  </div>    
+
+  <p>&nbsp;</p>
+    <div style="row">
+      <div class="col col-lg-12">
+      <h2>Step 2: Provision your Kubernetes Cluster</h2>
+          
+      <p>Provisioning your Kubernetes cluster is easy. Jenkins X does not really care how you provision your cluster, however there
+          are many resources that are provisioned, so we recommend using the Terraform Modules we've made available</p>
+          
+      <ul class="nav nav-tabs" id="provision-cluster-tabs">
+        <li class="nav-item">
+          <a class="nav-link active" data-toggle="tab" href="#gcp">GCP</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link disabled" data-toggle="tab" href="#aws">AWS (coming soon)</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link disabled" data-toggle="tab" href="#azure">Azure (coming soon)</a>
+        </li>
+      </ul>
+      <div id="provision-cluster-tabs" class="tab-content">
+        <div class="tab-pane fade show active" id="gcp">
+          <div class="row">
+            <div class="col-sm-4">
+                  <a href="/docs/getting-started/setup/create-cluster/google/"><img src="https://www.pulumi.com/logos/tech/gcp.svg"/></i></a>
+            </div>
+          </div>
+          <h4>Use GKE Terraform Module</h4>
+          <p>1 . Create a <strong>main.tf</strong> file and add the code snippet below.</p>
+{{<highlight javascript>}}
+module "jx" {
+  source  = "jenkins-x/jx/google"
```

Comment:
The example should be more like:

```
module "jx" {
  source  = "jenkins-x/jx/google"
  gcp_project = "<gcp-project-id>"
}
```

We should not use a fixed version. That gets out of date. By not specifying a version the latest version will be used. Also, we should explicitly list the required variables.